libs/sdk-ui-dashboard/src/model/commands/filters.ts

```diff
@@ -569,6 +575,55 @@ export function changeAttributeFilterSelection(
     };
 }
 
+/**
+ * Creates the ChangeAttributeFilterSelection command for adhoc migrated attribute filter.
+ *
+ * @remarks
+ * Dispatching this command will result in application of element selection for the dashboard attribute filter
+ * with the provided id, or error in case of invalid update (e.g. non-existing filterLocalId).
+ *
+ * The attribute elements can be provided either using their URI (primary key) or value. Together with the
+ * elements you must indicate the selection type - either 'IN' or 'NOT_IN'.
+ *
+ * @remarks see {@link resetAttributeFilterSelection} for convenience function to select all attribute elements of
+ *  particular filter.
+ *
+ * See also {@link applyAttributeFilter} for convenient function when you have an {@link @gooddata/sdk-model#IAttributeFilter} instance.
+ *
+ *  @example
+ * ```
+ * const selectionType = isPositiveAttributeFilter ? "IN" : "NOT_IN";
+ * ```
+ *
+ * To create this command without a need to calculate the payload values from a {@link @gooddata/sdk-model#IFilter} object,
+ * we recommend to use {@link applyAttributeFilter} command creator instead.
+ *
+ * @param filterLocalId - dashboard attribute filter's local id
+ * @param elements - elements
+ * @param selectionType - selection type. either 'IN' or 'NOT_IN'
+ * @param correlationId - specify correlation id to use for this command. this will be included in all
+ *  events that will be emitted during the command processing
+ *
+ * @internal
+ */
+export function changeMigratedAttributeFilterSelection(
+    filterLocalId: string,
+    elements: IAttributeElements,
+    selectionType: AttributeFilterSelectionType,
+    correlationId?: string,
+): ChangeAttributeFilterSelection {
+    return {
+        type: "GDC.DASH/CMD.FILTER_CONTEXT.ATTRIBUTE_FILTER.CHANGE_SELECTION",
+        correlationId,
+        payload: {
+            filterLocalId,
+            elements,
+            selectionType,
+            isResultOfMigration: true,
```

Comment:
This is the only change from the original command creator that creates the command with same payload. I could not use original one due to a optional parameter of the function at the end (I could not add another one to the end or precede it).